arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,776 @@
+          "description": "The connection endpoint for the DocumentDB database account.",
+          "type": "string",
+          "readOnly": true
+        },
+        "databaseAccountOfferType": {
+          "description": "The offer type for the DocumentDB database account. Default value: Standard.",
+          "readOnly": true,
+          "$ref": "#/definitions/DatabaseAccountOfferType"
+        },
+        "consistencyPolicy": {
+          "description": "The consistency policy for the DocumentDB database account.",
+          "$ref": "#/definitions/ConsistencyPolicy"
+        },
+        "writeLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the write location for the DocumentDB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "readLocations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains of the read locations enabled for the DocumentDB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "failoverPolicies": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the regions ordered by their failover priorities.",
+          "items": {
+            "$ref": "#/definitions/FailoverPolicy"
+          }
+        }
+      }
+    },
+    "DatabaseAccountCreateUpdateProperties": {
+      "description": "Properties to create and update Azure DocumentDB database accounts.",
+      "type": "object",
+      "properties": {
+        "consistencyPolicy": {
+          "description": "The consistency policy for the DocumentDB account.",
+          "$ref": "#/definitions/ConsistencyPolicy"
+        },
+        "locations": {
+          "type": "array",
+          "description": "An array that contains the georeplication locations enabled for the DocumentDB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        }
+      },
+      "required": [
+        "consistencyPolicy",
+        "locations"
+      ]
+    },
+    "DatabaseAccountCreateUpdateParameters": {
+      "description": "Parameters to create and update DocumentDB database accounts.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DatabaseAccountCreateUpdateProperties"
+        },
+        "databaseAccountOfferType": {
+          "$ref": "#/definitions/DatabaseAccountOfferType"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "required": [
+        "databaseAccountOfferType"
+      ]
+    },
+    "DatabaseAccountPatchParameters": {
+      "description": "Parameters for patching Azure DocumentDB database account properties.",
+      "type": "object",
+      "properties": {
```

Comment:
Why can you only update tags in the PATCH? PATCH too is one form of update. It seems weird that databaseAccount properties like location and consistencyPolicy can be updated via PUT (CreateOrUpdate) but cannot be updated via PATCH.